keras/backend/cntk_backend.py

```diff
@@ -1494,14 +1494,17 @@ def conv1d(x, kernel, strides=1, padding='valid',
         kernel = C.swapaxes(kernel, 0, 2)
 
     padding = _preprocess_border_mode(padding)
-    strides = [strides]
+
+    if dev.type() == 0 and dilation_rate != 1:
+        raise ValueError('Dilated convolution on CPU is not supported by CNTK backend. ' +
+                         'Please set dilation_rate with 1.')
```

Comment:
`'Please set dilation_date to 1. You passed: %d' % (dilation_rate,)`. And please use backquotes around code keywords.